src/clj/matcher_combinators/midje.clj

```diff
@@ -89,7 +96,9 @@
       (throw (ArityException. arg-count "throws-match")))))
 
 (checkers.defining/defchecker throws-match
```

Comment:
Does `defchecker` support metadata? Can we/do we need to add `^:deprecated` here too?

Reply:
Yes, as it turns out. Done.